ibm/service/power/resource_ibm_pi_network.go

```diff
@@ -79,7 +79,7 @@ func ResourceIBMPINetwork() *schema.Resource {
 				Optional:      true,
 				Computed:      true,
 				Deprecated:    "This field is deprecated, use pi_network_mtu instead.",
-				ConflictsWith: []string{helpers.PINetworkMtu},
+				ConflictsWith: []string{helpers.PINetworkMtu, helpers.PINetworkJumbo},
```

Comment:
Trying to add the same argument?

Reply:
Ah sorry I think this was a typo from the merge conflict.